linkup-cli/src/paid_tunnel.rs

```diff
@@ -138,13 +139,21 @@ impl PaidTunnelManager for RealPaidTunnelManager {
             account_id
         );
         let (client, headers) = prepare_client_and_headers(&RealSystem)?;
-        let query_url = format!("{}?name=tunnel-{}", url, tunnel_name);
+        let query_url = format!("{}?name={}", url, tunnel_name);
 
         let parsed: GetTunnelApiResponse = send_request(&client, &query_url, headers, None, "GET")?;
         if parsed.result.is_empty() {
             Ok(None)
         } else {
-            Ok(Some(parsed.result[0].id.clone()))
+            // Check if there exists a tunnel with this name that hasn't been deleted
```

Comment:
Deleting a tunnel doesn't delete it, it just sets "deleted_at". Instead, it needs to find the tunnel name and where "deleted_at" is null